dbt-metricflow/extra-hatch-configuration/requirements-cli.txt

```diff
@@ -1,5 +1,5 @@
 # Internal dependencies
-dbt-core>=1.8.0, <1.9.0
+dbt-core @ git+https://github.com/dbt-labs/dbt-core@e53420c1d073dc81609ae7aa84cef6ee09650576#subdirectory=core
```

Comment:
Argh, this means we cannot release dbt-metricflow until 1.9 rolls out. I'm so frustrated.